generative_ai/grounding.py

```diff
@@ -29,8 +32,8 @@ def grounding(
 
     from vertexai.language_models import GroundingSource, TextGenerationModel
 
-    # TODO(developer): Update values for project_id, location
-    vertexai.init(project=project_id, location=location)
+    # TODO(developer): Update project_id and location
+    vertexai.init(project=PROJECT_ID, location=LOCATION)
```

Comment:
Lets maintain consistency. Simple option is `vertexai.init(project=PROJECT_ID, location="us-central1")`

```suggestion
    vertexai.init(project=PROJECT_ID, location="us-central1")
```

Reply:
✅ Done